spec/features/merchants/items/index_spec.rb

```diff
@@ -7,9 +7,14 @@
     @merchant_3 = Merchant.create!(name: 'Tom Haverford')
     @merchant_4 = Merchant.create!(name: 'April Ludgate')
 
```

Comment:
I think there are some places in this test where we could remove test data. IE, I think this spec will run just fine with only one instance of merchant. It might be worth it to trim some of the unneeded objects out of our before(:each) statement.

Reply:
Removed extra merchant data.